src/main/java/green/projectile/Projectile.java

```diff
@@ -0,0 +1,30 @@
+package green.projectile;
+
+public class Projectile {
+    final double BASE_CASE = 0;
+    final double GRAVITY = 9.8;
+    final double HALF = 0.5;
+    double angle, velocity, seconds = BASE_CASE;
```

Comment:
In Java, these variables are automatically initialized to 0 so you don't need to do this.

Also break this into multiple lines.